YAFCui/Core/Structs.cs

```diff
@@ -44,7 +44,16 @@ public enum SchemeColor {
         Green,
         GreenAlt,
         GreenText,
-        GreenTextFaint
+        GreenTextFaint,
+        // Tagged row colors
+        TagColorGreenBackground,
+        TagColorGreenText,
```

Comment:
So, instead of

```csharp
TagColorGreenBackground,
TagColorGreenText,
```

it would be

```csharp
TagColorGreen,
TagColorGreenAlt, // unused
TagColorGreenText,
TagColorGreenTextFaint, // unused
```
?

Did I get it right? If yes, I'm not sure if this is a good way to introduce these new colors :-/

Reply:
> Did I get it right?

Yes

`TagColorGreenAlt` could/should be (ab)used for alternating rows I guess? (See some other comment thread).
But no need if you feel that alternating colors is not desired!

>  I'm not sure if this is a good way to introduce these new colors 

Agreed, but this is the current YAFC coloring system...

For the Magenta color (used for the overproduction feature) I only use the 'main' tag, the other three are not used directly, only by the coloring system of YAFC...

I guess we could take a good look at it and improve it in a future PR. There seems to be a lot to gain/simplify here.

Reply:
Ok, thank you. For now, I'll introduce the correct color quadruples for red, green, blue, and yellow highlighting.

Reply:
The colors for tagged rows are now provided in the same four quadruples like the other colors.

Your comment started with only this aspect. I'd say we continue the discussion about colors further down the line.